lib/conjur/conjur_config.rb

```diff
@@ -63,5 +71,15 @@ def trusted_proxies_valid?
     rescue
       false
     end
+
+    def authenticators_valid?
+      authenticators.each do |authenticator|
+        return false unless /^(authn|authn-.+)$/.match?(authenticator.strip)
+      end
+      
+      true
```

Comment:
You should be able to replace this with `#all?`: https://apidock.com/ruby/Enumerable/all%3F

```suggestion
      authenticators.all? do |authenticator|
        /^(authn|authn-.+)$/.match?(authenticator.strip)
      end
```